rotcon/convolve.py

```diff
@@ -64,6 +64,9 @@ def __init__(self, *args, **kwargs):
               1.5 Å are reasonable. The default is conservative. [``2.2`` Å]
            *histogramBins*
              tuple ``(dmin, dmax, delta)`` in Ångström [``(0.0, 100.0, 1.0)``]
+           *useNOelectron*
```

Comment:
This should be a boolean. Internally, this should be either `True` or `False`. 

Argparse or optparse have a mechanism to handle flags. Then the user should be able to just say
```
---useNOelectron
```

or
```
---no-useNOelectron
```